addons/istio/istio.yaml

```
@@ -42,14 +34,9 @@ spec:
   chartReference:
     chart: istio
     repo: https://mesosphere.github.io/charts/staging
-    version: 1.6.12
+    version: 1.8.1
```

```suggestion
    version: 1.8.2
```
@GoelDeepak should we bump this to latest version?